sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi_simple.c

```diff
@@ -340,9 +340,17 @@ static inline void stabilization_indi_calc_cmd(int32_t indi_commands[], struct I
   indi.u_in.r = indi.u[2].o[0] + indi.du.r;
 
   //bound the total control input
+#ifdef INDI_FULL_AUTHORITY
```

Comment:
This new option should be documented in the module xml....

Reply:
Also please use `#if FLAG` instead of `#ifdef FLAG` according to http://docs.paparazziuav.org/latest/stylec.html#stylecpp